vscode-trace-webviews/src/trace-explorer/time-range/vscode-time-range-data-widget.tsx

```diff
@@ -66,20 +62,20 @@ class TimeRangeDataWidget extends React.Component {
 
     private _onVscodeSelectionRangeUpdated = (data: any): void => {
         if (data) {
-            const result = JSONBig.parse(data);
-            signalManager().emit('SELECTION_RANGE_UPDATED', result);
+            signalManager().emit('SELECTION_RANGE_UPDATED', JSONBigUtils.parse(data));
         }
     };
 
     private _onVscodeViewRangeUpdated = (data: any): void => {
         if (data) {
-            signalManager().emit('VIEW_RANGE_UPDATED', JSONBig.parse(data));
+            signalManager().emit('VIEW_RANGE_UPDATED', JSONBigUtils.parse(data));
         }
     };
 
     private _onVscodeRestoreView = (data: any): void => {
         if (data) {
-            const { mapArray, activeData } = JSONBig.parse(data);
+            const { mapArray, activeData } = JSONBigUtils.parse<{mapArray: Array<ExperimentTimeRangeData>, activeData: ExperimentTimeRangeData}>(data);
+            // TODO add schema because ExperimentTimeRangeData include bigints
```

Comment:
Is it the plan to address this TODO as part of this PR? If not, add it as a follow-up in PR description?

Reply:
Yes, I can address it.